ml4bio/ml4bio.py

```diff
@@ -2222,17 +2241,47 @@ def initUI(self):
         self.rocRadioButton.toggled.connect(self.plot)
         self.prRadioButton.toggled.connect(self.plot)
 
-        # global geometry
-        self.resize(1064, 700)
         self.setWindowTitle('ML4Bio (version {})'.format(ml4bio.__version__))
-        self.leftPanel.resize(360, 680)
-        self.leftPanel.setStyleSheet("QStackedWidget {background-color:rgb(226, 226, 226)}")
-        self.leftPanel.move(10, 10)
-        self.rightPanel.resize(680, 680)
-        self.rightPanel.move(380, 10)
+
+        self.leftPanel.resize(0.34*w, h*0.95)
+        self.leftPanel.move(0.01*w, 0.02*h)
+        self.rightPanel.resize(0.60*w, 0.95*h)
+        self.rightPanel.move(0.36*w, 0.02*h)
 
         self.show()
 
+
+    def resizeEvent(self, event):
+        """
+        Catches the resize event to set things up
+        """
+        w = self.width()
+        h = self.height()
+        self.leftPanel.resize(0.34*w, h*0.95)
+        self.leftPanel.move(0.01*w, 0.02*h)
+        self.rightPanel.resize(0.60*w, 0.95*h)
+        self.rightPanel.move(0.36*w, 0.02*h)
+
+        #Right panel bottom left table
+        self.model_summary_.setMaximumWidth(w*0.28)
+        self.model_summary_.setColumnWidth(0, w*0.145)
+
+
+        #We have to not plot too small or we get a crash
+        #TODO: Store these min vals somewhere?
```

Comment:
If we keep this, can you please open a new issue to track this behavior?  I'm comfortable merging, but we should use an issue to note the TODOs in the longer term.

Reply:
I removed the TODO and instead note the issue (which I will open post merge). I just want to note the strange error passing behavior.